test/domain_bridge/test_domain_bridge.cpp

```diff
@@ -60,42 +62,53 @@ TEST_F(TestDomainBridge, get_options)
 
 TEST_F(TestDomainBridge, bridge_topic_valid)
 {
-  // Nominal test
+  // Individual parameters
   {
     domain_bridge::DomainBridge bridge;
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 2);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 2u);
+  }
+  // Struct parameter
+  {
+    domain_bridge::DomainBridge bridge;
+    domain_bridge::TopicBridge topic_bridge = {
+      "foo",
+      "test_msgs/msg/BasicTypes",
+      1u,
+      2u
+    };
+    bridge.bridge_topic(topic_bridge);
   }
   // Same domain ID
   {
     domain_bridge::DomainBridge bridge;
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 1);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 1u);
   }
   // Topic with namespace
   {
     domain_bridge::DomainBridge bridge;
-    bridge.bridge_topic("foo/bar/baz", "test_msgs/msg/BasicTypes", 1, 2);
+    bridge.bridge_topic("foo/bar/baz", "test_msgs/msg/BasicTypes", 1u, 2u);
   }
   // Multiple topics
   {
     domain_bridge::DomainBridge bridge;
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 2);
-    bridge.bridge_topic("bar", "test_msgs/msg/BasicTypes", 1, 2);
-    bridge.bridge_topic("baz", "test_msgs/msg/BasicTypes", 1, 2);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 2u);
+    bridge.bridge_topic("bar", "test_msgs/msg/BasicTypes", 1u, 2u);
+    bridge.bridge_topic("baz", "test_msgs/msg/BasicTypes", 1u, 2u);
   }
   // Same topic, different domains
   {
     domain_bridge::DomainBridge bridge;
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 2);
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 3);
-    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 2, 1);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 2u);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 3u);
+    bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 2u, 1u);
   }
   // Same topic, different types
   // This use-case isn't clearly supported by ROS 2, see https://github.com/ros2/ros2/issues/1095
   // {
   //   domain_bridge::DomainBridge bridge;
-  //   bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1, 2);
-  //   bridge.bridge_topic("foo", "test_msgs/msg/Nested", 1, 2);
-  //   bridge.bridge_topic("foo", "test_msgs/msg/Constants", 1, 2);
+  //   bridge.bridge_topic("foo", "test_msgs/msg/BasicTypes", 1u, 2u);
+  //   bridge.bridge_topic("foo", "test_msgs/msg/Nested", 1u, 2u);
+  //   bridge.bridge_topic("foo", "test_msgs/msg/Constants", 1u, 2u);
   // }
```

Comment:
this test case can just be removed

Reply:
I want to document that this undefined behavior (or maybe simply guard against it by declaring it invalid). I'll address this is a follow up PR